src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,30 @@
+const ADD_BOOK = 'bookstore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
+
+const initialState = {
+  books: [],
+};
+
+export const addBook = (payload) => ({
+  type: ADD_BOOK,
+  payload,
+});
+
+export const removeBook = (payload) => ({
+  type: REMOVE_BOOK,
+  payload,
+});
+
+const bookSlice = (state = initialState, action) => {
+  switch (action.type) {
+    case ADD_BOOK:
+      return [...state, action.payload];
+
+    case REMOVE_BOOK:
+      return state.filter((bookItem) => bookItem.id !== action.payload);
+    default:
+      return state;
+  }
+};
+
+export default bookSlice;
```

Comment:
- [ ] - [ ] Please kindly implement this using redux toolkit (RTK) according to the project requirement, it seems like you're using the old method. ℹ️ 

<img width="791" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/230048704-13d254c9-90e9-407e-b7d5-19c1464cb418.png">